targets/kodi

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+REQUIRES='x11'
+DESCRIPTION='Installs the KODI media player. (Approx. 140MB)'
+HOSTBIN='startkodi'
+CHROOTETC='kodi-keyboard.xml kodi-cycle.py'
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+if [ "$DISTRO" = 'debian' ]; then
```

Comment:
We shouldn't be using jessie-backports in sid, should we?  The right condition would then be `if release -le jessie; then`, which will apply only to debian releases jessie or earlier.